tests/unittests/UIKit/UIFontTests.mm

```diff
@@ -108,8 +108,8 @@
 
 TEST(UIFont, Metrics) {
     float size = 22.0f;
-    UIFont* font = [UIFont fontWithName:@"Segoe UI Semibold Italic" size:size];
-    ASSERT_OBJCEQ(@"Segoe UI Semibold Italic", [font fontName]);
+    UIFont* font = [UIFont fontWithName:@"Segoe UI" size:size];
+    ASSERT_OBJCEQ(@"Segoe UI", [font fontName]);
 
     ASSERT_NE(0, [font descender]);
```

Comment:
If it's time sensitive, understood.

---
In reply to: [85244713](https://github.com/Microsoft/WinObjC/pull/1234#discussion_r85244713) [](ancestors = 85244713,85243923)